plans/feat-2573-remote-host-calendar-update-delete.md

```
@@ -0,0 +1,86 @@
+# feat(remote-host): `google.calendar.updateEvent` / `deleteEvent` (#2573)
+
+## Background
+
+#2572 added calendar/task update + delete to the `google` **tool**. The sweep
```

markdownlint-cli2 (0.23.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)